mne/io/tests/test_raw.py

```diff
@@ -38,6 +38,7 @@ def _test_raw_reader(reader, test_preloading=True, **kwargs):
         # don't assume the first is preloaded
         buffer_fname = op.join(tempdir, 'buffer')
         picks = rng.permutation(np.arange(len(raw.ch_names)))[:10]
+        picks = np.append(picks, len(raw.ch_names) - 1)  # test trigger channel
```

Comment:
I added this for making sure that the constructed stim channels are also tested. Almost made a bug because this wasn't here earlier.